src/Common/src/System/Security/Cryptography/Asn1V2.cs

```diff
@@ -398,7 +398,11 @@ public override int GetByteCount(string s)
             return GetByteCount(s.AsReadOnlySpan());
         }
 
-        private int GetByteCount(ReadOnlySpan<char> chars)
+        public
+#if netcoreapp
+            override
```

Comment:
Are these really only virtual in netcoreapp? Or do they just not exist on netcoreapp? I assume these will not work on anything but netcoreapp or are we using the System.Memory OOB on those platforms?

Reply:
On netstandard they don't exist, so I can't override; but on netcoreapp they do, and they're virtual.

Previously this file had only compiled under netstandard.